src/Service/Installment.php

```diff
@@ -117,11 +117,11 @@ public function updateInstallmentsInArrears($today)
            if($today > $installmentDueDate)
            {
                 $next = $this->findNext($installment);
-                $this->updateWithInterest($next,
-                    $installment->getValue());
-               
-                $installment->setStatus(
-                    $this->installmentStatusRepository->getInArrears());
+                if ($next) {
+                    $this->updateWithInterest($next, $installment->getValue());
+                }
```

Comment:
This condition was added to avoid a bug: when there were not next installments, an exception was being thrown :boom: